packages/shared/src/components/widgets/SocialShare.tsx

```diff
@@ -72,27 +71,16 @@ export const SocialShare = ({
   const onSharedSuccessfully = () => {
     trackEvent(postAnalyticsEvent(AnalyticsEvent.ShareToSquad, post));
 
-    if (onSquadShare) {
-      onSquadShare();
+    if (onClose) {
+      onClose();
     }
```

Comment:
We could do `onClose?.();` in typescript instead to convert this to a 1-liner.

Reply:
Used to do that in the past, but I received comments about having it like this (even though what you suggest is basically this and also what I prefer). Let me write these cases down in a note to have it discussed on a bigger forum in the future.